docs/DafnyRef/Types.md

```diff
@@ -420,7 +421,19 @@ But `11` is not a valid `bv3` literal.
 OrdinalType_ = "ORDINAL"
 ````
 
-TO BE WRITTEN
+Values of type `ORDINAL` behave like `nat`s in many ways, with one important difference:
+there is an `ORDINAL` value that is larger than any `nat`.
+- a value of type `nat` may be explicitly converted to an `ORDINAL` using `as ORDINAL`
+- a value of type `ORDINAL` may be explicitly converted to a `nat` using `as nat` if it is known to be less than some `nat` value 
+- non-negative numeric literals may be considered `ORDINAL` literals
+- `ORDINAL`s may be compared, using `== != < <= > >=`
+- two `ORDINAL`s may be added and the result is `>=` either one of them
+- two `ORDINAL`s may be subtracted if the LHS is `>=` the RHS (so the result is non-negative) and the RHS value is equal to some `nat` value. That is, for `ORDINAL` `x`, `assert x - x == 0;` fails
+ but `assert x < 1000000000 ==> x-x == 0;` succeeds.
+- `*`, `/` and `%` are not defined for `ORDINAL`s
+- there is no literal in Dafny that represents the `ORDINAL` value that is larger than any `nat` (typically written $\omega$ in mathematics)
+
+In Dafny, `ORDINAL`s are used primarily in conjunction with extreme functions and lemmas.
```

Comment:
Can you please add a link to extreme functions and lemmas?

Reply:
Done